### DISCUSSION TITLE
API question regarding AribaIntegrationToolkit replacement iFlow

### DISCUSSION THREAD

**Discussion by watwatwhat** [SAP-samples/cloud-integration-flow#88] [Q&A]
Hi experts,
I'm looking for information about the API endpoint "https://[ariba-host]/Buyer/fileupload" and https://[ariba-host]/Buyer/filedownload" which is used in aribaintegrationtoolkit provided here; apibusinesshub-integration-recipies/Recipes/for/saparibaintegrationtoolkit.
I searched the SAP Business Accelerator Hub and Ariba Developers Portal, but I couldn't find any detailed information regarding this API. What I found was that Ariba provides Open REST APIs and SOAP Web Services, but it seems that the APIs in question are not contained in documents of either API services.
Is this unofficial API? If not, could you suggest where to get a clue to reveal this API?

**Comment by mporrasDole** [comment]
Hi all, Any update?
Regards
